### PR TITLE
adding unarchiving guide

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Adding a guide on how to unarchive things to the docs.

### CODE REVIEW

File: docs/unarchiving.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+directory:
+
+```bash
+echo 'CONTENT_ROOT=/path/to/content/files' >> .env
+echo 'CONTENT_ARCHIVED_ROOT=/path/to/archived-content/files' >> .env
+echo 'CONTENT_TRANSLATED_ROOT=/path/to/translated-content-rendered/files'
+ >> .env
+```
+
+Note: If an .env file does not already exist, it will be created by the first of
```
Comment (peterbe) [inline docs/unarchiving.md:47]:
```suggestion
Note: If an `.env` file does not already exist, it will be created by the first of
```

**Review by peterbe — APPROVED** [review]
Awesome!

**Comment by peterbe** [comment]
@escattone I'd be happy to merge. Wanna take a quick skim. 
I told Chris to not worry about perfecting this documentation but a brain dump is MUCH better than nothing.

**Comment by chrisdavidmills** [comment]
cool, cheers for making the fix @peterbe. Feel free to merge it when you are happy.

**Review by escattone — APPROVED** [review]
Thanks @chrisdavidmills !

**Pull request merged**